src/Cake.Common/Tools/DotNet/DotNetAliases.cs

```diff
@@ -2883,5 +2884,78 @@ public static DotNetPackageList DotNetListPackage(this ICakeContext context, str
             var lister = new DotNetPackageLister(context.FileSystem, context.Environment, context.ProcessRunner, context.Tools);
             return lister.List(project, settings);
         }
+
+        /// <summary>
```

Comment:
Does it make sense to consider using partial classes for this DotNetAliases to better grouping and navigation?
e.g.  
DotNetAliases.cs
- DotNetAliases.Sln.cs
- DotNetAliases.Package.cs
- ...
What do you think?

Reply:
Yes, not a bad idea.

Reply:
in scope of this PR or create a new one? what do you think?

Reply:
separate PR is probably more clear.